build/wix/main.wxs

```diff
@@ -157,6 +157,18 @@
       <ComponentRef Id="PathUser" />
       <ComponentRef Id="PathSystem" />
     </Feature>
+    <UI>
+       <UIRef Id="WixUI_RD" />
+       <Publish Dialog="ExitDialog"
+         Control="Finish"
+         Event="DoAction"
+         Value="LaunchApplication"
+       >WIXUI_EXITDIALOGOPTIONALCHECKBOXTEXT = 1 and NOT Installed</Publish>
```

Comment:
Did you mean for the condition to refer to `WIXUI_EXITDIALOGOPTIONALCHECKBOX` instead of `WIXUI_EXITDIALOGOPTIONALCHECKBOXTEXT`? (That is, do this if the checkbox is checked, instead of doing it if the prompt text is defined.)

My understanding is that `0` and `{}` (i.e. undefined) are falsy, and everything else (`1`, a random GUID, etc…) are true, so this could just be `WIXUI_EXITDIALOGOPTIONALCHECKBOX AND NOT Installed`.

Reply:
Good catch. Naturally the app was running because I always checked that box.